### PR TITLE
ANC - skip Obstetric History step 2 on first pregnancy

### PR DESCRIPTION
**Pull request by anvmn**
#1325

### CODE REVIEW

**Comment by anvmn** [comment]
Peek 2024-09-19 23-08

File: ci-scripts/circleci_wait.sh [hunk 1]
```diff
@@ -3,7 +3,7 @@
 
 CMD="$*"
 
-TIMEOUT=3700
+TIMEOUT=4300
```
Comment (AronNovak) [inline ci-scripts/circleci_wait.sh:6]: Was it a one-time hiccup that the timeout wasn't enough or do you think it's a frequent problem?

Comment (anvmn) [reply ci-scripts/circleci_wait.sh:6]: @AronNovak 

It seems that whenever we exceed 1h execution, we get timeout anyways.
So, this change is not effective. Saw tasks fail with this increased timeout.
All failing tasks state 1h execution time. For example: 

https://app.circleci.com/pipelines/github/TIP-Global-Health/eheza-app/4742/workflows/022cde6a-0985-44b5-9143-90f92554c68b/jobs/16322

Selection_875

Comment (AronNovak) [reply ci-scripts/circleci_wait.sh:6]: @anvmn if needed, let's open an issue for me and I can investigate a bit. Thanks for the details!

**Review by AronNovak — APPROVED** [review]

File: ci-scripts/circleci_wait.sh (hunk 1 above)
Comment (anvmn) [reply ci-scripts/circleci_wait.sh:6]: @AronNovak 

https://github.com/TIP-Global-Health/eheza-app/issues/1335

**Pull request merged**